values/gitea/gitea.gotmpl

```diff
@@ -0,0 +1,51 @@
+{{- $v := .Environment.Values }}
+{{- $o := $v.oidc }}
+{{- $g := $v.charts | get "gitea" dict }}
+{{- $k := $v.charts | get "keycloak" dict }}
+{{- $hasKeycloak := $k | get "enabled" true }}
+{{- $realm := $k | get "realm" "master" }}
+{{- $keycloakIssuer := printf "https://keycloak.%s/realms/%s" $v.cluster.domain $realm }}
+
+{{- $giteadomain := printf "gitea.%s" $v.cluster.domain}}
+{{- $addIgnorePolicyAnnotation := readFile "../../helmfile.d/utils/addIgnorePolicyAnnotation.gotmpl" }}
+
+nameOverride: gitea
+fullnameOverride: gitea
+# clusterDomain: {{ $v.cluster.domain }}
+
+gitea:
+  podAnnotations: 
+    {{-  tpl $addIgnorePolicyAnnotation (dict "policies" (list "psp-allowed-users"))  | nindent 4 }}
+  admin:
+    username: otomi
+    password: {{ $g | get "admin.password" "bladibla" }}
+  config:
+    server:
+      DOMAIN: {{ $giteadomain }}
+      ROOT_URL: "https://%(DOMAIN)s/"
+    openid:
+      ENABLE_OPENID_SIGNIN: true
+  metrics:
+    enabled: true
+    serviceMonitor:
+      enabled: true
+  oauth:
+    enabled: true
+    name: {{ $hasKeycloak | ternary $k.idp.alias "otomi" }}
+    provider: openidConnect
+    key: {{ $hasKeycloak | ternary $k.idp.clientID $o.clientID }}
+    secret: {{ $hasKeycloak | ternary $k.idp.clientSecret $o.clientSecret }}
+    autoDiscoverUrl: '{{ $hasKeycloak | ternary $keycloakIssuer $o.issuer }}/.well-known/openid-configuration'
+
+persistence:
+  size: 1Gi
+
+postgresql:
+  persistence:
+    size: 1Gi
+
+# Can't be used (for now) as gitea requires root in the docker image
```

Comment:
why?

Reply:
Gitea 1.13 docker image requires root to run. When adding security context an issue arises that is documented:
https://gitea.com/gitea/helm-chart/pulls/115

For Gitea 1.14 a rootless image will be produced, but it's currently only available as: `latest-rootless` and does not set the version. Making it less consistent overall

Reply:
Let's stick to stable version.

Reply:
Isn't that already merged to master last year? https://github.com/go-gitea/gitea/pull/10154

What is hindering you from using that?

Reply:
I see they are late with stuff: https://github.com/go-gitea/gitea/issues/14819

resolving this